gr-blocks/lib/float_to_complex_impl.cc

```diff
@@ -65,10 +64,13 @@ namespace gr {
 	break;
 
       case 2:
+      {
 	//for (size_t j = 0; j < noutput_items*d_vlen; j++)
 	//  out[j] = gr_complex (r[j], i[j]);
```

Comment:
One could actually remove the commented lines in a separate commit.

Reply:
yep, but let's not do that pre-merge, should we?